tests/puppeteer/pages/Install/install.js

```diff
@@ -0,0 +1,152 @@
+const CommonPage = require('../commonPage');
+
+module.exports = class INSTALL extends CommonPage {
+  constructor(page) {
+    super(page);
+
+    // Define Step Titles
+    this.firstStepFrTitle = 'Bienvenue sur l\'installateur de PrestaShop';
+    this.firstStepEnTitle = 'Welcome to the PrestaShop';
+    this.secondStepEnTitle = 'License Agreements';
+    this.thirdStepEnTitle = 'We are currently checking PrestaShop compatibility with your system environment';
+    this.fourthStepEnTitle = 'Information about your Store';
+    this.fifthStepEnTitle = 'Configure your database by filling out the following fields';
+    this.finalStepEnTitle = 'Your installation is finished!';
+
+    // Selectors for all steps
+    this.nextStepButton = '#btNext';
+
+    // Selectors for step 1
+    this.firstStepPageTitle = 'h2';
+    this.languageSelect = '#langList';
+
+    // Selectors for step 2
+    this.secondStepPageTitle = 'h2#licenses-agreement';
+    this.termsConditionsCheckbox = '#set_license';
+
+    // Selectors for step 3
+    this.thirdStepPageTitle = '#sheet_system h2';
+    this.stepFinishedLeftMenu = '#leftpannel #tabs li.finished:nth-child(3)';
```

Comment:
Please add the `type` of the element

Reply:
updated to thirdStepFinishedListItem.
Thanks